website/lang/spanish.js

```diff
@@ -1,155 +1,153 @@
+},
+searchBar: {
+  zipTextField: {
+    header: "Buscar citas cerca de",
+    placeholder: "Ingrese un código postal de 5 dígitos",
   },
-  news: {
-    date: "2021/23/03",
-    message:
-      "Otra vez <strong>Walgreens</strong> ha comenzado a bloquear el acceso, sino que espero que las últimas soluciones deberían restaurar la disponibilidad de citas de Walgreens (nuevamente). Mientras tanto, yo tambien he agregado citas de <strong>Centura Health</strong> para usuarios de Colorado.",
-    suffix:
-      "Cualquier comentario es bienvenido: <a href='m&#97;ilto&#58;v%&#54;1&#99;&#99;&#105;ne&#64;nic&#107;%6D&#46;org'>vacc&#105;ne&#64;ni&#99;k&#109;&#46;o&#114;&#103;</a > o <a href='https://twitter.com/nickblah'>@nickblah</a>. ¡Y si eres un programador y tienes interés en ayudar en <a href='https://github.com/GUI/covid-vaccine-finder/issues'>GitHub</a>, eso sería estupendo (aunque el código sigue siendo complicado)!",
+  radius: {
+    header: "Dentro de",
+    anyDistance: "Cualquier distancia",
+    xDistance: "{distance} millas",
   },
-  scanningDetails: {
-    scanning:
-      "Escaneando {chain_count} farmacias ({store_count} tiendas diferentes) en {state}",
+  withoutAppointments: "Mostrar ubicaciones sin citas actuales",
+  button: "Buscar",
+  noResults:
+    "Ahora no podemos encontrar citas abiertas para su búsqueda. Intente expandir su búsqueda o verifique nuevamente más tarde (las citas pueden aparecer y desaparecer rápidamente).",
+},
+buttons: {
+  newAppointments: "Consultar nuevas citas",
+  visitWebsite: "Visitar el sitio de web de {name}",
+},
+share: {
+  button: { on: "Compartir en {name}", by: "Compartir por {name}" },
+  sites: {
+    facebook: "Facebook",
+    twitter: "Twitter",
+    tumblr: "Tumblr",
+    email: "Correo electrónico",
+    pinterest: "Pinterest",
+    linkedIn: "LinkedIn",
+    reddit: "Reddit",
+    whatsApp: "WhatsApp",
+    hackerNews: "Hacker News",
   },
-  searchBar: {
-    zipTextField: {
-      header: "Buscar citas cerca de",
-      placeholder: "Ingrese un código postal de 5 dígitos",
-    },
-    radius: {
-      header: "Dentro de",
-      anyDistance: "Cualquier distancia",
-      xDistance: "{distance} millas",
-    },
-    withoutAppointments: "Mostrar ubicaciones sin citas actuales",
-    button: "Buscar",
-    noResults:
-      "Ahora no podemos encontrar citas abiertas para su búsqueda. Intente expandir su búsqueda o verifique nuevamente más tarde (las citas pueden aparecer y desaparecer rápidamente).",
+},
+map: {
+  legend: {
+    available: "Citas disponibles recientemente",
+    notAvailable: "Citas no disponibles",
+    unknown: "Se desconoce el estado de la cita",
   },
-  buttons: {
-    newAppointments: "Consultar nuevas citas",
-    visitWebsite: "Visitar el sitio de web de {name}",
+},
+time: {
+  unknown: "Hora desconocida",
+},
+appointments: {
+  available: "Citas disponibles a partir de {date_time}",
```

Comment:
This {date_time} shouldn't be here. I think your branch is a couple of days out of date?